src/SearchClient.php

```
@@ -54,6 +55,10 @@ public static function createWithConfig(SearchConfig $config)
     {
         $config = clone $config;
 
+        if (!$config->getAppId() || !$config->getApiKey()) {
+            throw new UnreachableException('Missing config: appId or apiKey. Please check your settings');
```

could you throw a `\InvalidArgumentException` here instead of `UnreachableException`?